dashboard/test/fixtures/resource.yml

```diff
@@ -17,4 +17,10 @@ resource_104:
   id: 104
   key: resource_104
   name: class site
-  url: testsebsite.fake
+  url: testwebsite.fake
+resource_105:
+  id: 105
+  key: resource_105
+  name: class resource
+  url: testresource.fake
+  course_version: course_version_2019
```

Comment:
thinking aloud... it doesn't look like these get generated by dashboard/scripts/generate_fixtures.rb. but, it also looks like tests would keep passing if someone were to run that script, because (1) these wouldn't get erased, and (2) there is no link between these fixtures (course version or resources) and the generated ones which would get broken. so I think we are ok for now, but it is also possible we are going to have to start generating these sometime soon if scripts and unit group fixtures start to depend on their course version id. Does that sound right?

Reply:
As long as someone does not add course_version or resource to the generate fixtures script than they should not get erased when the script is run I think.

Reply:
Yeah I thought about adding them to the generate fixtures script but decided to take the approach of not doing that until it's needed. If we know we're going to need to add course version into the generate fixtures script, then I can do that.